packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketInfo/CoinmarketInfoItem.tsx

```diff
@@ -29,7 +29,7 @@ export const CoinmarketInfoItem = ({
         {type === 'exchange' || isReceive ? (
             <Column alignItems="flex-end" gap={spacings.xxxs}>
                 <Row gap={spacings.xs}>
-                    <CoinmarketCoinLogo cryptoId={currency!} size={20} />
+                    {currency && <CoinmarketCoinLogo cryptoId={currency} size={20} />}
                     <CoinmarketCryptoAmount amount={amount} cryptoId={currency!} />
```

Comment:
Should we handle CoinmarketCryptoAmount's currency as well?
```suggestion
                    {currency && <CoinmarketCryptoAmount amount={amount} cryptoId={currency} />}
```

Reply:
Nice catch, fixed